frontend/src/metabase-types/store/app.ts

```diff
@@ -17,4 +17,5 @@ export interface AppBreadCrumbs {
 export interface AppState {
   errorPage: AppErrorDescriptor | null;
   isNavbarOpen: boolean;
+  isDndAvailable: boolean;
```

Comment:
@heypoom this is the boolean thing you were asking about, instead of using a separate context provider